cinema_service/urls.py

```diff
@@ -2,10 +2,21 @@
 from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import path, include
+from drf_spectacular.views import (
+    SpectacularAPIView,
+    SpectacularRedocView,
+    SpectacularSwaggerView
+)
+
 
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
     path("api/user/", include("user.urls", namespace="user")),
     path("__debug__/", include("debug_toolbar.urls")),
+    path("api/cinema/schema/", SpectacularAPIView.as_view(), name="schema"),
+    path("api/cinema/doc/swagger/",
+         SpectacularSwaggerView.as_view(url_name="schema"), name="swagger-ui"),
+    path("api/cinema/doc/redoc/",
+         SpectacularRedocView.as_view(url_name="schema"), name="redoc"),
```

Comment:
actually swagger shouldn't be a part of any app, thus you have to remove cinema from the endpoints. it's the best practice how to access swagger for frontend developers